app/src/main/java/com/it/hungvt/movieapp/screen/detail/DetailViewModel.java

```diff
@@ -0,0 +1,93 @@
+package com.it.hungvt.movieapp.screen.detail;
+
+import android.content.Context;
+import android.content.Intent;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.widget.Toast;
+import com.it.hungvt.movieapp.data.model.Movie;
+import com.it.hungvt.movieapp.BR;
+import com.it.hungvt.movieapp.screen.home.HomeActivity;
```

Comment:
WARNING: [UnusedImports] Avoid unused imports such as 'com.it.hungvt.movieapp.screen.home.HomeActivity'
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#UnusedImports